crates/turborepo-lib/src/cli/mod.rs

```diff
@@ -202,9 +202,6 @@ pub struct Args {
     pub check_for_update: bool,
     #[clap(long = "__test-run", global = true, hide = true)]
     pub test_run: bool,
-    /// Enable the experimental UI
-    #[clap(long, hide = true, global = true)]
```

Comment:
Just to confirm, we're not leaving a flag to toggle off the UI?

Reply:
Correct, if desired we'll add one.